components/JumboSection.vue

```diff
@@ -0,0 +1,56 @@
+<template>
+  <section class="hero is-large">
+    <div class="hero-body" :style="{ backgroundImage: 'url(' + image + ')' }">
+      <div class="container">
+        <h1 class="title" v-html="title">
+          {{ title }}
+        </h1>
+      </div>
+    </div>
+    <!-- <div
+      class="centered"
+      :style="{
+        backgroundImage: 'url(' + image + ')',
+        height: imageHeight + 'px'
+      }"
+    >
+      <div
+        class="title"
+        :style="{ 'padding-top': paddingTop + 'px' }"
+        v-html="title"
+      ></div>
+    </div> -->
+  </section>
+</template>
+
+<script>
+export default {
+  props: {
+    title: {
+      type: String,
+      required: true
+    },
+    imageHeight: {
+      type: null,
+      required: true
+    },
```

Comment:
```js
imageHeight: {
      type: null,
      required: true
    },
```
Type null doesn't make sense. It does not mean 'Number' but 'Null' or 'Nothing'.